src/Components/CycleManagement/CycleClassifications.jsx

```diff
@@ -0,0 +1,104 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { withRouter } from 'react-router';
+import TabbedCard from 'Components/TabbedCard';
+import BackButton from 'Components/BackButton';
+import ProfileSectionTitle from 'Components/ProfileSectionTitle/ProfileSectionTitle';
+import { cycleClassificationsFetchData } from 'actions/cycleManagement';
+
+const CycleClassifications = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(cycleClassificationsFetchData());
+  }, []);
+
+  const results = useSelector(state => state.cycleClassificationsData);
+  const isLoading = useSelector(state => state.cycleClassificationsIsLoading);
+
+  const [classifications, setClassifications] = useState(results?.cycle_classifications ?? []);
+  const [selections, setSelections] = useState(results?.classification_selections ?? []);
+
+  const pc = results?.cycle_classifications ?? [];
+  const cs = results?.classification_selections ?? [];
+
+  useEffect(() => {
+    setClassifications(pc);
+    setSelections(cs);
+  }, [results]);
+
+  // const handleSelection = (code, event) => {
+  //   const newSelections = selections.map(s => {
+  //     if (s.code === code) {
+  //       return {
+  //         ...s,
+  //         value: event.target.checked ? '1' : '0',
+  //       };
+  //     }
+  //     return s;
+  //   });
+  //   setSelections(newSelections);
+  // };
+
+  const cycleClassCard = () => (isLoading ?
+    <div className="loading-animation--5">
+      <div className="loading-message pbl-20">
+        Loading additional data
+      </div>
+    </div> :
+    (selections.map(sel => (
+      <div className="position-classifications">
+        <div className="line-separated-fields">
+          <div>
+            <span>{sel.cycle_name} - {sel.cycle_desc}</span>
+          </div>
+        </div>
+        <div className="table-container">
+          <table>
+            <thead>
+              <tr>
+                {classifications?.map((o) => (
+                  <th key={o.code}>{o.short_description}</th>
+                ))}
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                {classifications?.map((c) => (
+                  <td key={c.code}>
+                    <input
+                      type="checkbox"
+                      name={c.code}
+                      checked={sel?.values.find(s => c.code === s.code && s.value === '1') ?? false}
```

Comment:
```suggestion
                      checked={sel?.values.find(s => c.code === s.code).value === '1'}
```
optional. find it based on code and then check value for === 1